tools/coqdep/lib/args.ml

```diff
@@ -67,22 +89,19 @@ let parse st args =
     | "-vos" :: ll -> parse { st with vos = true } ll
     | ("-noglob" | "-no-glob") :: ll -> parse { st with noglob = true } ll
     | "-noinit" :: ll -> (* do nothing *) parse st ll
-    | "-f" :: f :: ll -> parse { st with coqproject = Some f } ll
-    | "-I" :: r :: ll -> parse { st with ml_path = r :: st.ml_path } ll
+    | "-f" :: f :: ll -> parse (add_from_coqproject st f) ll
+    | "-I" :: r :: ll -> parse (add_ml_path st r) ll
     | "-I" :: [] -> usage ()
     | "-R" :: r :: ln :: ll ->
-       let ln = if String.equal ln "Coq" then "Stdlib" else ln in
-       parse { st with vo_path = (true, r, ln) :: st.vo_path } ll
+       parse (add_vo_path st (true, r, ln)) ll
     | "-Q" :: r :: ln :: ll ->
-       let ln = if String.equal ln "Coq" then "Stdlib" else ln in
-       parse { st with vo_path = (false, r, ln) :: st.vo_path } ll
-    | "-R" :: ([] | [_]) -> usage ()
+       parse (add_vo_path st (false, r, ln)) ll
+    | ("-Q"|"-R") :: ([] | [_]) -> usage ()
     | "-exclude-dir" :: r :: ll -> System.exclude_directory r; parse st ll
     | "-exclude-dir" :: [] -> usage ()
     | "-coqlib" :: r :: ll -> Boot.Env.set_coqlib r; parse st ll
     | "-coqlib" :: [] -> usage ()
     | "-dyndep" :: dyndep :: ll -> parse { st with dyndep } ll
-    | "-m" :: m :: ll -> parse { st with meta_files = st.meta_files @ [m]} ll
```

Comment:
Maybe you want to emit a deprecation / warning message for users of `-m` ?

Reply:
I don't think so